lighthouse-core/test/results/sample_v2.json

```diff
@@ -4356,30 +4466,31 @@
     },
     "is-crawlable": {
       "score": null,
-      "displayValue": "",
       "rawValue": null,
-      "error": true,
-      "debugString": "Audit error: Required RobotsTxt gatherer did not run.",
+      "displayValue": "",
+      "errorMessage": "Audit error: Required RobotsTxt gatherer did not run.",
+      "warnings": [],
       "scoreDisplayMode": "error",
       "name": "is-crawlable",
       "description": "Page is blocked from indexing",
       "helpText": "Search engines are unable to include your pages in search results if they don't have permission to crawl them. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/indexing)."
     },
     "robots-txt": {
       "score": null,
-      "displayValue": "",
       "rawValue": null,
-      "error": true,
-      "debugString": "Audit error: Required RobotsTxt gatherer did not run.",
+      "displayValue": "",
+      "errorMessage": "Audit error: Required RobotsTxt gatherer did not run.",
+      "warnings": [],
```

Comment:
can we leave warnings undefined if there are none?
same thing with displayValue, tbh. :)

Reply:
yeah sgtm 👍 @brendankenny agree?

Reply:
@brendankenny agree? yeah, sounds good